### ISSUE TITLE
Proxies issues

### ISSUE THREAD

**Issue by Bass66** [V4NSH4J/discord-mass-DM-GO#419]
Hello,
I get this error when i put the proxies in input txt file and i try to test my tokens checker "error while getting necessary data couldn't initialize token "
I tried a lot of proxies and i have the same problem.

**Comment by V4NSH4J** [comment]
Are you able to use the program without proxies?

**Comment by Bass66** [comment]
Yes i do the program is working fine without proxies

**Comment by V4NSH4J** [comment]
And in proxies.txt you have put proxies in format user:pass@ip:port or ip:port (ip can be interchanged with hostname) and ensured proxy_from_file is enabled?

**Comment by Bass66** [comment]
Yes i'm sure of the format and also i enabled proxy from file but still don't work , i sent you a message in telegram to talk about this prob

**Comment by V4NSH4J** [comment]
What's your @?

**Comment by Bass66** [comment]
@bascrypt666

**Comment by V4NSH4J** [comment]
Solved, was using incorrect proxy format. Make a new issue if you have more questions!

**Comment by lucas666888** [comment]
> 已解決，使用了不正確的代理格式。如果您有更多問題，請提出新問題！

Do you have a recommended agent? Do you have contact information? I have something to talk to you about cooperation

**Comment by APrimaveraV** [comment]
Hi from Dominican Republic sr
